components/library/zlib/Makefile

```diff
@@ -28,18 +28,17 @@
 BUILD_BITS= 32_and_64
 include ../../../make-rules/shared-macros.mk
```

Comment:
Please add
USE_COMMON_TEST_MASTER= no
here and update the test results in the test folder. You can find the new results in build/test/{i86|amd64}/results-{32|64}.snapshot after running gmake test.